CHANGELOG.rst

```diff
@@ -17,6 +17,11 @@ Unreleased
 [3.1.0] - 2021-04-28
```

Comment:
Should not it be changed to the latest version and date?

Reply:
@shimulch, of course! :facepalm: